projects/mysadna/src/app/components/user-details/user-details.component.html

```diff
@@ -4,16 +4,18 @@
     <div class="user-container media-row-col">
       <div class="user-label">
         <div class="flex">
-          <img [src]="user.getUserImg()">
-          <div class="user-name">{{ user.getUserName() }}</div>
+          <img [src]="user.getImageUrl()">
+          <div class="user-name">{{ user.getId() }}</div>
           <div class="email">{{ user.getEmail() }}</div>
           <div class="social-networks">
-            <a
-              href="https://github.com/hasadna/hasadna"
-              *ngFor="let socialNetwork of user.getSocialNetworkList()"
-            >
-              <img src="./assets/images/{{socialNetwork}}.svg">
-            </a>
+            <div class="social-network" *ngFor="let socialNetwork of user.getSocialNetworkList()">
+              <a [href]="socialNetwork.getUrl()">
+                <img [src]="'./assets/images/' + types[socialNetwork.getType()] + '.svg'">
+              </a>
+            </div>
+          </div>
+          <div class="cv-url">
+            <a href="https://my-first-firebase-projec-be6b0.firebaseapp.com/cv-template.html/">CV File</a>
```

Comment:
It's not a mistake, but why absolute link?
Why not:
```html
<a href="cv-template.html/">CV File</a>
```
Thing is, if we'll decide to deploy app on different domain (e.g. [https://mysadna-profile-pages.firebaseapp.com](https://mysadna-profile-pages.firebaseapp.com))
We don't need to change the link.

Reply:
Also, I think the link is incorrect. It should be
```
assets/cv-pages/cv-template.html
```
isn't it?

Reply:
I'll try that.